From 8b4ce3db96309c537dbc18b1c4d59fb9c2576492 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Tue, 28 Mar 2017 22:16:40 +0100
Subject: [PATCH 13/32] progress: To satisfy CBM requirements, only use /boot/
 not /boot/efi

This will ensure that cbm_is_mounted("/boot") is satisfed internally within
clr-boot-manager when chrooting, allowing the failure to detect a root
device to actually work and skip past the mount path.

This provides initial testing towards ikeydoherty/clr-boot-manager#54

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 os_installer2/pages/progress.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/os_installer2/pages/progress.py b/os_installer2/pages/progress.py
index f62f818..7336c14 100644
--- a/os_installer2/pages/progress.py
+++ b/os_installer2/pages/progress.py
@@ -244,7 +244,7 @@ class InstallerProgressPage(BasePage):
         """ Mount the ESP into the root """
         self.set_display_string("Mounting EFI System Partition")
         root = self.get_installer_target_filesystem()
-        fpath = os.path.join(root, "boot/efi")
+        fpath = os.path.join(root, "boot")
         if not os.path.exists(fpath):
             try:
                 os.makedirs(fpath, mode=0o755)
-- 
2.13.2

